src/Goteo/Application/Config.php

```diff
@@ -255,6 +255,8 @@ static private function setDirConfiguration() {
 		Payment::addMethod('Goteo\Payment\Method\PaypalPaymentMethod');
 		// Adding Cash non-public payment method (manual admin investions)
 		Payment::addMethod('Goteo\Payment\Method\CashPaymentMethod', true);
+		// Adding Stripe payment method
+		Payment::addMethod('Goteo\Payment\Method\StripeSubscriptionPaymentMethod');
```

Comment:
This change requires that the `settings.yml` has the key `stripe`  inside the `payments` section. We should update also the documentation to make sure that it is by default in every application. If not, we can change it and only add it if the stripe payment is active (But it will change completely the current configuration of the payments)